README.md

```diff
@@ -19,3 +19,94 @@ Frontend Service can be found here -`https://github.com/Emergency-Response-Demo/
   `oc new-app --name find-service --strategy docker https://github.com/Emergency-Response-Demo/find-service`
   
   You will need to pass `INCIDENT_SERVICE_URL` and `MISSION_SERVICE_URL` as Environment variable.
+  
+## Deploying using Tekton Pipeline
+
+* Prerequisite
+
+    OpenShift v4.x Cluster. You can install using `OpenShift Pipelines Operator` from the Operator Hub.
+    
+    You will also use the Tekton CLI (tkn). Download the Tekton CLI by following [instructions](https://github.com/tektoncd/cli#installing-tkn) available on the CLI GitHub repository.
+
+* Installation Steps
+
+    1. Fork this repository so that you can create a trigger for any GitHub Event which will trigger the pipeline and deploy the new code.
+    
+    2. Clone the repository to your local machine.
+    
+        ` git clone https://github.com/<your-github-username>/find-service`
+    
+    3. Traverse to the pipeline folder.
+    
+        ` cd find-service/pipeline/`
+        
+    4. Login to your OpenShift Cluster.
+    
+    5. Create a new project `find-my-relative`
+    
+        ` oc new-project find-my-relative`
+    
+    6. Install the Pipeline Resources,Task and Pipeline.
+    
+        1. Pipeline - There are two task in pipeline - buildah and openshift-client.
+        
+            Install pipeline using below command - 
+        
+            ` oc apply -f 02-pipelines/ `
```

Comment:
For the commands, it might be better to use code blocks instead of inline backticks.(https://guides.github.com/features/mastering-markdown/#examples)